nixos/modules/services/ttys/getty.nix

```diff
@@ -92,17 +92,6 @@ in
         '';
       };
 
-      serialSpeed = mkOption {
```

Comment:
How would one set serial speed than?

Reply:
`stty -F /dev/ttyUSB0 115200`
or for non standard baudrate you can do
`boot.kernelParams = ["console=ttyS2,1500000"]`
according to https://github.com/NixOS/nixpkgs/pull/128079

Reply:
```
  imports = [
    (mkRemovedOptionModule [ "services" "getty" "serialSpeed" ] "set non-standard baudrates with `boot.kernelParams` i.e.  boot.kernelParams = ["console=ttyS2,1500000"];")
  ];
```

Reply:
Done. Thanks for the suggestion.